firmware/steering/kia_soul_ps/steering_control_module.ino

```diff
@@ -386,6 +426,9 @@ static void process_ps_ctrl_steering_command( const uint8_t * const rx_frame_buf
     current_ctrl_state.commanded_steering_angle = control_data->steering_wheel_angle_command / 9.0 ;
 
     current_ctrl_state.steering_angle_rate_max = control_data->steering_wheel_max_velocity * 9.0;
+    current_ctrl_state.steering_angle_rate_max = 350;
```

Comment:
Should we be setting this manually for users? Is this necessary for steering wheel detector to work?